examples/data-objects/table-tree/src/Table/tablePropTypes.ts

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { DragEvent } from "react";
+
+import { Column, Row } from "./tableSchema.js";
+
+export interface TableRowViewProps {
+	row: Row;
```

Comment:
I think all of the properties in both of these types can be made `readonly`, which I would recommend. The array properties can further be made into readonly arrays. E.g. `readonly columns: readonly Column[];`